src/views/RetailerView/RetailerView.jsx

```diff
@@ -7,15 +7,8 @@ import { fetchOneRetailer, actionUpdateRetailer } from "../../actions";
 import Dashboard from "../../components/Dashboard/Dashboard.jsx";
 
 function RetailerView(props) {
-  //checks local storage for key
-  let checkStorage = localStorage.getItem("currentCapacity");
-  const maxCapacity = JSON.parse(localStorage.retailer).maxCapacity;
-  let initialCount;
-  if (checkStorage) {
-    initialCount = parseInt(checkStorage);
-  } else {
-    initialCount = 0;
-  };
+  const [maxCapacity] = useState(JSON.parse(localStorage.retailer).maxCapacity)
+  const [initialCount] = useState(parseInt(localStorage.getItem("currentCapacity")) || 0)
   const [custCount, setCustCount] = useState(initialCount);
```

Comment:
the more I look at this the more I'm thinking it belongs in a componentDidMount useEffect().  Let's discuss.